app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -0,0 +1,24 @@
+@import 'twitter-bootstrap-static/bootstrap';
+@import 'twitter-bootstrap-static/fontawesome';
+
+body {
+  background: #f3f4f5;
```

Comment:
Color literals like `#f3f4f5` should only be used in variable declarations; they should be referred to via variable everywhere else.